### ISSUE TITLE
Disco day may 2024 - explore new date picker for VZV

### ISSUE THREAD

**Issue by roseeichelmann** [cityofaustin/atd-data-tech#17132]
We need to find a new date picker module for vzv! 

Options:
- [react-datepicker](https://www.npmjs.com/package/react-datepicker). This is what we use as our datepicker in VZE. Would make sense to use the same datepicker across the VZ suite. I tested installing it in VZV with the latest version 6.9.0, but im getting this [error](https://github.com/date-fns/date-fns/issues/3635)... it will only work if we revert to an older version like 2.9.6 bc of our super outdated `create-react-app` version... seems kinda not great?
    - Follow up - I upgraded our CRA react-scripts version to 5.0 so that I could use the latest version of react-datepicker and also fixes some of our npm vulnerabilities... then i ran into [this error](https://www.alchemy.com/blog/how-to-polyfill-node-core-modules-in-webpack-5) bc of a package we have that requires pollyfills, `hookrouter`, which has an archived repo :skull: so perhaps we should migrate to [react-router](https://github.com/Paratron/hookrouter) which is actually maintained ORR
    -  What if we [migrate VZV to Next.js](https://nextjs.org/docs/app/building-your-application/upgrading/from-create-react-app)??? Would solve all of these problems! ^
        - Nextjs has routing built in
        - Its better :) 

To recap: this discovery day started with trying to find a new date picker package and snowballed into me upgrading CRA and then suggesting we migrate away from CRA to Nextjs or some other framework that will be better for our use case
